src/components/dashboard/main-content/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -503,93 +516,109 @@ const BaseCourseCard = ({
     if (!courseMiscText) {
       return null;
     }
+
     return (
-      <small className="mb-0 mt-2">
-        {courseMiscText}
-      </small>
+      <div className="small">
+        {intl.formatMessage(courseMiscText, {
+          a: getCoursePaceHyperlink,
+          pacing: `${pacing}`,
```

Comment:
[nit/curious] Is the string literal here needed if the value is just `pacing`? I.e., could this be simplified to:

```jsx
{intl.formatMessage(courseMiscText, {
  a: getCoursePaceHyperlink,
  pacing,
})}
```

Reply:
Yep good catch, remnant of a previous string construction here 👍🏽 ****